### ISSUE TITLE
Newsletter - H1 tag missing

### ISSUE THREAD

**Issue by cathysarisky** [TryGhost/Ghost#29612]
### Issue Summary

The newsletter body uses `<h2>` tags for the subtitles (good), but is lacking a `<h1>` for the top-level title of the newsletter, eg "Roots of belonging", that one is marked up as regular text. 

## Related WCAG
- [SC 1.3.1 Info and Relationships (A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Steps to Reproduce

Send a newsletter.  Inspect to reveal that the title, while styled as large text, is not marked up as H1. 

### Ghost Version

Ghost Pro

### Node.js Version

Ghost Pro

### How did you install Ghost?

Ghost Pro

### Database type

MySQL 8

### Browser & OS version

N/A

### Relevant log / error output

```shell

```

### Code of Conduct

- [x] I agree to be friendly and polite to people in this repository
